code/_onclick/click.dm

```diff
@@ -499,13 +496,6 @@ if(selected_ability.target_flags & flagname && !istype(A, typepath)){\
 	SEND_SIGNAL(src, COMSIG_CLICK_CTRL_SHIFT)
 
 
-/*
-	Ctrl+Middle click
-*/
-/atom/proc/CtrlMiddleClickOn(atom/A)
-	SEND_SIGNAL(src, COMSIG_CLICK_CTRL_MIDDLE, A)
-
```

Comment:
???
why are you removing this??

Reply:
The only action using control middle mouse click is the queen overwatch selection.

Leaving it in would be dead code. Doing nothing.

If someone wants to add it back in for another feature or action, they are welcome to do so. Same with any other combination.

Reply:
Oh, I didnt delete the comsig declaration. I'll need to do that tomorrow